guides/common/modules/proc_altering-the-privilege-elevation-method.adoc

```diff
@@ -0,0 +1,20 @@
+[id="altering-the-privilege-elevation-method_{context}"]
+= Altering the Privilege Elevation Method
+
+By default, push-based remote execution uses `sudo` to switch from the SSH user to the effective user that executes the script.
+In some situations, you may require to use another method, such as `su` or `dzdo`.
+You can globally configure an alternative method to switch the effective user in your {Project} settings.
+
+.Prerequisite
+
+* If you want to use `dzdo` for Ansible jobs, you have to install the `community.general` Ansible collection, which contains the *dzdo* Ansible plugin.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/solutions/7025758[Running Ansible job on hosts registered with Red Hat Satellite 6 fails with error could not find matching plugin: 'dzdo'] in the _Red{nbsp}Hat Knowledgebase_.
+endif::[]
+
+.Procedure
+
+. Navigate to *Administer* > *Settings* > *Remote Execution*.
```

Comment:
I am unsure about "X > Y > Z" vs "X > Y" + "On the Z tab, click ...". I believe we currently have both versions within the docs. Any general thoughts? @adamlazik1 @asteflova 

For this instance: your call Lena.

Reply:
I've decided to add the tab selection as an additional but separate step.

Reply:
> I am unsure about "X > Y > Z" vs "X > Y" + "On the Z tab, click ...". I believe we currently have both versions within the docs. Any general thoughts? @adamlazik1 @asteflova

A bit off-topic (let me know if you want to talk about this elsewhere), but I've been wondering lately: To what extent do you as a team strive to be consistent in these things? Is it expected to stick to the same wording across all content? Or just within a single guide? Within an assembly? Within a module?

Reply:
Due to the modularity, I strive for consistency for everything within `guides/common/modules/`. If there is already some inconsistency (like 30:70) in the docs, then I don't strongly recommend going with the majority. If it's more like 5:95, then I always recommend to go with the 95%. I'd rather handle such cases in separate PRs.

Always remember: the docs is what we make it; conventions and consistency is human-made by us. :v: